var/spack/repos/builtin/packages/hiprand/package.py

```diff
@@ -22,6 +22,8 @@ class Hiprand(CMakePackage, CudaPackage, ROCmPackage):
 
     version("develop", branch="develop")
     version("master", branch="master")
+    version("5.6.1", commit="379e077db067e7b5fd4c27446f6e6f6b1c92a263")
```

Comment:
There is now a tag for hipRAND. It lags behind the other libraries at the moment, but it should be part of the normal tagging process starting with ROCm 6.0.

Reply:
Updated with the 5.6 tag